diff --git a/.gitmodules b/.gitmodules
index 7bc2effd3..6cd4532fb 100644
--- ./.gitmodules
+++ ./.gitmodules
@@ -1,9 +1,9 @@
 [submodule "engine"]
 	path = engine
-	url = https://gitlab.com/minds/engine.git
+	url = git@github.com:huangyingw/engine.git
 [submodule "front"]
 	path = front
-	url = https://gitlab.com/minds/front.git
+	url = git@github.com:huangyingw/front.git
 [submodule "sockets"]
 	path = sockets
-	url = https://github.com/minds/sockets.git 
+	url = git@github.com:huangyingw/sockets.git
diff --git a/docker-compose.yml b/docker-compose.yml
index 6efe4a908..d98e3d0c0 100644
--- ./docker-compose.yml
+++ ./docker-compose.yml
@@ -24,6 +24,11 @@ services:
 
   ## APP ENGINE
 
+  minds_php:
+    build:
+      context: ./engine/containers/php
+    image: minds/php:7.3
+
   php-fpm:
     build:
       context: ./engine
diff --git a/local/commands/install.js b/local/commands/install.js
index fd363c7ee..e5ca40a6a 100644
--- ./local/commands/install.js
+++ ./local/commands/install.js
@@ -24,26 +24,8 @@ module.exports.handler = async argv => {
     scope.push('local stack');
   }
 
-  const prompt = await prompts([
-    {
-      type: 'toggle',
-      name: 'confirm',
-      message: `This will WIPE: [${scope.join(', ')}], if exists. Proceed?`,
-      active: 'Yes',
-      inactive: 'No'
-    }
-  ]);
-
-  console.log('');
-
-  if (!prompt.confirm) {
-    console.log('Cancelled by user');
-    process.exit(1);
-  }
-
   const tasks = new Listr([
     require('../tasks/stop'),
-    argv.front && require('../tasks/cleanup-front'),
     argv.front && require('../tasks/build-front'),
     argv.stack && require('../tasks/cleanup-stack'),
     argv.stack && require('../tasks/provision-elasticsearch'),
diff --git a/local/tasks/build-containers.js b/local/tasks/build-containers.js
index 3bfad8d0e..036a2ae3f 100644
--- ./local/tasks/build-containers.js
+++ ./local/tasks/build-containers.js
@@ -2,5 +2,5 @@ const doco = require('../lib/doco');
 
 module.exports = {
   title: 'Building containers',
-  task: () => doco('build', '--no-cache')
+  task: () => doco('build')
 };
