diff --git a/.gitmodules b/.gitmodules
index 7bc2effd3..6cd4532fb 100644
--- ./.gitmodules
+++ ./.gitmodules
@@ -1,9 +1,9 @@
 [submodule "engine"]
 	path = engine
-	url = https://gitlab.com/minds/engine.git
+	url = git@github.com:huangyingw/engine.git
 [submodule "front"]
 	path = front
-	url = https://gitlab.com/minds/front.git
+	url = git@github.com:huangyingw/front.git
 [submodule "sockets"]
 	path = sockets
-	url = https://github.com/minds/sockets.git 
+	url = git@github.com:huangyingw/sockets.git
diff --git a/docker-compose.with-phpspec.yml b/docker-compose.with-phpspec.yml
index df5b038c1..c5b2f1415 100644
--- ./docker-compose.with-phpspec.yml
+++ ./docker-compose.with-phpspec.yml
@@ -11,3 +11,5 @@ services:
       - keys:/.dev
     networks:
       - app
+    depends_on:
+      - minds_php
diff --git a/docker-compose.yml b/docker-compose.yml
index 6efe4a908..ae1701f6f 100644
--- ./docker-compose.yml
+++ ./docker-compose.yml
@@ -24,12 +24,18 @@ services:
 
   ## APP ENGINE
 
+  minds_php:
+    build:
+      context: ./engine/containers/php
+    image: minds/php:7.3
+
   php-fpm:
     build:
       context: ./engine
       dockerfile: ./containers/php-fpm/Dockerfile.dev
     mem_limit: 512MB
     depends_on:
+      - minds_php
       - cassandra
       - mongo
       - elasticsearch
@@ -47,6 +53,7 @@ services:
       context: ./engine
       dockerfile: ./containers/php-runners/Dockerfile
     depends_on:
+      - minds_php
       - cassandra
       - elasticsearch
       - rabbitmq
@@ -102,7 +109,6 @@ services:
   elasticsearch:
     image: elasticsearch:6.6.2
     entrypoint: /set-permissions-and-start.sh
-    mem_limit: 512MB # keep an eye
     ulimits:
       nproc: 65536
       memlock:
@@ -149,8 +155,8 @@ services:
       - ./engine/:/var/www/Minds/engine:delegated
       - keys:/.dev
     depends_on:
-      cassandra:
-        condition: service_healthy
+        - cassandra
+        - minds_php
 
   elasticsearch-legacy:
     image: elasticsearch:5.6-alpine
diff --git a/local/commands/install.js b/local/commands/install.js
index 6c28c8069..a4d98b2ae 100644
--- ./local/commands/install.js
+++ ./local/commands/install.js
@@ -14,23 +14,6 @@ module.exports.handler = async argv => {
     console.log('\nWARNING: Close any tool that might be watching Minds folder (e.g. VSCode, TortoiseGit, etc.)\n');
   }
 
-  const prompt = await prompts([
-    {
-      type: 'toggle',
-      name: 'confirm',
-      message: 'This will WIPE your local stack, if exists. Proceed?',
-      active: 'Yes',
-      inactive: 'No'
-    }
-  ]);
-
-  console.log('');
-
-  if (!prompt.confirm) {
-    console.log('Cancelled by user');
-    process.exit(1);
-  }
-
   const tasks = new Listr([
     require('../tasks/stop'),
     require('../tasks/cleanup'),
diff --git a/local/tasks/build-containers.js b/local/tasks/build-containers.js
index 3bfad8d0e..036a2ae3f 100644
--- ./local/tasks/build-containers.js
+++ ./local/tasks/build-containers.js
@@ -2,5 +2,5 @@ const doco = require('../lib/doco');
 
 module.exports = {
   title: 'Building containers',
-  task: () => doco('build', '--no-cache')
+  task: () => doco('build')
 };
