diff --git a/.gitmodules b/.gitmodules
index 7bc2effd3..50737f864 100644
--- ./.gitmodules
+++ ./.gitmodules
@@ -1,9 +1,9 @@
+[submodule "sockets"]
+	path = sockets
+	url = git@github.com:huangyingw/sockets.git
 [submodule "engine"]
 	path = engine
-	url = https://gitlab.com/minds/engine.git
+	url = git@github.com:huangyingw/engine.git
 [submodule "front"]
 	path = front
-	url = https://gitlab.com/minds/front.git
-[submodule "sockets"]
-	path = sockets
-	url = https://github.com/minds/sockets.git 
+	url = git@github.com:huangyingw/front.git
